### PR TITLE
New package: beard-0.3

### PR DESCRIPTION
**Pull request by RecursiveForest**
Simple hibernation daemon https://github.com/ahills/beard

### CODE REVIEW

File: srcpkgs/beard/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'beard'
+pkgname=beard
+version=0.3
+revision=1
+short_desc="Simple hibernation daemon"
+maintainer="Samantha Baldwin <fuhsaz+void@cryptic.li>"
+license="Public domain"
```
Comment (Vaelatern) [inline srcpkgs/beard/template]: Add `noarch=yes`

Comment (Vaelatern) [inline srcpkgs/beard/template]: `license="Unlicense"`

Comment (RecursiveForest) [inline srcpkgs/beard/template]: Done, thanks.

File: srcpkgs/beard/template [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+revision=1
+noarch=yes
+short_desc="Simple hibernation daemon"
+maintainer="Samantha Baldwin <fuhsaz+void@cryptic.li>"
+license="Unlicense"
+homepage="https://github.com/ahills/beard"
+distfiles="$homepage/archive/v$version.tar.gz"
+checksum=00b6d3f62f20d2fa79b86c0260696fb0db9d75402a7444680672a12c16365b70
+
+depends="runit-void"
```
Comment (ebfe) [inline srcpkgs/beard/template]: that's part of base-system

Comment (RecursiveForest) [inline srcpkgs/beard/template]: Understood, thanks. I assume the right thing to do is just remove the `depends` line alltogether.

**Pull request closed without merging**